src/aoapplication.cpp

```diff
@@ -146,8 +146,17 @@ void AOApplication::add_favorite_server(int p_server)
   server_type fav_server = server_list.at(p_server);
 
   QString str_port = QString::number(fav_server.port);
+  QString network_type;
+  switch (fav_server.socket_type) {
+  default:
+    fav_server.socket_type = TCP;
+  case TCP:
+    network_type = "TCP";
+  case WEBSOCKETS:
+    network_type = "WS";
```

Comment:
why does the write code use uppercase but the read code uses lowercase

Reply:
Sorry didn't read the read code will fix it.